framework/config/config.py

```diff
@@ -57,7 +58,18 @@ def __init__(self, root_dir, owtf_pid):
         # Available profiles = g -> General configuration, n -> Network plugin
         # order, w -> Web plugin order, r -> Resources file
         self.initialize_attributes()
-        self.LoadFrameworkConfigFromFile(os.path.join(self.RootDir, 'framework', 'config', 'framework_config.cfg'))
+        path = os.path.join(os.path.expanduser("~"), os.path.join('.owtf', 'configuration'))
+        try:
+            os.makedirs(path)
+        except OSError:
+            if not os.path.isdir(path):
+                raise
+        path_config = os.path.join(path, 'framework_config.cfg')
+        if not os.path.isfile(path_config):
+            with open(path_config,"w+") as f:
+                f.close()
+            shutil.copy(os.path.join(self.RootDir, 'configuration', 'framework_config.cfg.default'), path_config)
```

Comment:
Use `shutil.copyfile` instead. You will not require the above command to create a file beforehand.